tests/ut/language/parser/test_dump_tag_dsl.py

```
@@ -0,0 +1,229 @@
+            ) -> pl.Tensor[[16, 16], pl.FP32]:
+                pl.dump_tag(self.kernel)  # not a tensor Var
+                d = self.kernel(a, d)
+                return d
+
+        _ = P
+
+
+def test_dump_tag_rejects_wrong_arity() -> None:
+    """``pl.dump_tag()`` and ``pl.dump_tag(a, b)`` both fail at the
+    statement-position interceptor — exactly one positional arg is required."""
+    with pytest.raises(ParserSyntaxError, match="dump_tag.*exactly one positional"):
```

Ruff (0.15.14)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)